lib/prawn/svg/transform_utils.rb

```diff
@@ -0,0 +1,37 @@
+module Prawn::SVG::TransformUtils
```

Comment:
Stylistically I try to stay away from "Utils" in names, what do you think about `PDFMatrix` instead?

Reply:
Fine by me. Updated.